src/api/v1/users/schema.ts

```diff
@@ -1,5 +1,45 @@
 /*
  * Route validation schema for User routes
  */
-export { query } from '../schema/request';
+import * as Joi from 'joi';
+
+
+export { query, gender, id } from '../schema/request';
 export { response } from '../schema/response';
+
+const currentYear = (new Date()).getFullYear();
```

Comment:
Would this be better as a function? Otherwise we will forever be forever stuck in 2018...